projects/storefrontapp-e2e-cypress/cypress/helpers/notification.ts

```diff
@@ -1,17 +1,43 @@
 import { login } from './auth-forms';
+import { waitForPage } from './checkout-flow';
 import { generateMail, randomString } from './user';
 
 export const normalProductCode = '872912';
 export const firstProductCodeSelector =
   'cx-my-interests .cx-product-interests-product-item:first .cx-code';
 export const firstProductAscending = '4205431';
 export const firstProductDescending = '898520';
+export const NOTIFICATION_PREFERENCES_CHANGE_ENDPOINT_ALIAS =
+  'notificationPreferencesChange';
 
-// notification preference
 export function navigateToNotificationPreferencePage() {
```

Comment:
Please remove exports from the functions, that are only used internally in this file.